### ISSUE TITLE
RT2800USB Wifi USB Driver (RT5572)

### ISSUE THREAD

**Issue by meditant** [Shubhamvis98/kali-pinephone#75]
Hi,
I have some RT5572 WiFi USB adapters, and they work fine on Kali NetHunter Pro with the PinePhone Pro. However, the RT2800USB driver doesn’t exist in the SDM845 version of the kernel.
I tried compiling a desktop version, but it didn’t work, as I’m unable to find the source for SDM845.
Does anyone have any ideas?

Best regards,

**Comment by Shubhamvis98** [comment]
Have you changed the dr_mode in dts before trying?
And you can get the sdm845 kernel source from mobian repository.

**Comment by meditant** [comment]
Yes other adapter work, i look for sdm845 kernel sources from mobian.
Thanks

**Comment by Picters** [comment]
> Have you changed the dr_mode in dts before trying? And you can get the sdm845 kernel source from mobian repository.

Can you provide a link to the nethunter pro kernel sources?
